### ISSUE TITLE
Consider not requiring pytest in requirements.txt

### ISSUE THREAD

**Issue by cdent** [cdent/gabbi#230]
It should perhaps only be required to install pytest if pytest is going to be used, in which case, imports should be conditional?

I'm not sure it is that much of a big deal, but people do tend to prefer limiting requirements.

**Comment by jd** [comment]
You could use flavors.

**Comment by cdent** [comment]
> You could use flavors.

Yeah. I'm not feeling super motivated by this one, but would happily accept patches if someone was.
